.github/workflows/docs.yml

```diff
@@ -0,0 +1,31 @@
+name: mobile_docs
+
+on:
+  push:
+    branches:
+    - main
+  pull_request:
+
+jobs:
+  docs:
+    runs-on: ubuntu-20.04
+    timeout-minutes: 20
+    container:
+      image: envoyproxy/envoy-build-ubuntu:b0ff77ae3f25b0bf595f9b8bba46b489723ab446
+    steps:
+    - uses: actions/checkout@v1
+    - name: Add safe directory
+      run: git config --global --add safe.directory /__w/envoy/envoy
```

Comment:
use `GITHUB_WORKSPACE` instead of hardcoding?

Reply:
Seeing if `checkout@v3` removes the need for this.

Reply:
if it doesnt you can specify `safe.directory` with v3 directly i believe

Reply:
Still needed it but your `GITHUB_WORKSPACE` suggestion worked.